src/lib/autocomplete/autocomplete.spec.ts

```diff
@@ -17,18 +17,27 @@ import {
   MdAutocomplete,
   getMdAutocompleteMissingPanelError,
 } from './index';
+import {
+  ENTER,
+  DOWN_ARROW,
+  SPACE,
+  UP_ARROW,
+  ESCAPE,
+  Directionality,
+  Direction,
+  RxChain,
+  map,
+  startWith
+} from '@angular/cdk';
```

Comment:
Hmm, now that I'm seeing these imports, I'm thinking we need to change the `cdk` to use imports like
```ts
import {LiveAnnouncer} from '@angular/cdk/a11y';
```
Otherwise the library is too much of a grab-bag of unrelated stuff (especially for the aliased rxjs operators and keycodes)

Thoughts? It might be easier to make such a change first rather than to unroll the imports once they're all `@angular/cdk`